google/cloud/CMakeLists.txt

```diff
@@ -65,6 +65,8 @@ configure_file(internal/build_info.cc.in internal/build_info.cc)
 
 # the client library
 add_library(google_cloud_cpp_common
+            error.h
```

Comment:
Please rename these files to `terminate_handler.h` and `terminate_handler.cc`.

Reply:
Done